node_helper.js

```diff
@@ -30,60 +25,59 @@ module.exports = NodeHelper.create({
 		this.getHelper().sendSocketNotification("WETTERONLINE_RESULTS", event);
 	},
 
-	findGid: function(body) {
+	findGid (body) {
 		const exp = /WO\.geo = (\{[^\}]+\})/m;
 	
 		const match = body.match(exp);
 	
 		if(match) {
 			let obj = this.parseInlineJson(match[1]);
-			return obj['gid'];
+			return obj["gid"];
 		}
 	
 		throw new Error("city's gid could not be extracted");
 	},
 
-	extractEvent: function(dailyData, hourlyData, body) {
+	extractEvent (dailyData, hourlyData, body) {
 		// extract current temp
-		let currentTempMatch = body.match(/<div id="nowcast-card-temperature"[^>]*>.*?<div class="value">(\d+)<\/div>/ms);
+		let currentTempMatch = body.match(/<div id="nowcast-card-temperature"[\s\S]*?<div class="value">(-?\d+)<\/div>/ms);
```

Comment:
I don't remember why I adapted the first part 😔 I have now undone that.

And yes, the part with the minus is the important part. Otherwise it doesn't work with negative temperatures.